impeller/tools/impeller.gni

```diff
@@ -195,6 +200,26 @@ template("embed_blob") {
   }
 }
 
+# Dispatches to the build or prebuilt impellerc depending on the value of
+# the impeller_use_prebuilt_impellerc argument. Forwards all variables to
+# compiled_action_foreach or action_foreach as appropriate.
+template("_impellerc") {
+  if (impeller_use_prebuilt_impellerc == "") {
+    compiled_action_foreach(target_name) {
+      forward_variables_from(invoker, "*")
+      tool = "//flutter/impeller/compiler:impellerc"
+    }
+  } else {
+    action_foreach(target_name) {
```

Comment:
Ok to ignore this for this patch.

impellerc may not be stable from revision to revision. We'll want a nice failure mode if/when someone ends up using an incompatible impellerc with the current engine build.

Reply:
Added more text to the help message on the `gn` script flag.